### PR TITLE
Add QoS configurations for Quicksilver

### PR DESCRIPTION
**Pull request by rick-arista**
#### Why I did it
- Adds QoS configs for Quicksilver SKUs

##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305
- [x] 202311

#### Description for the changelog

Add QoS configuration files for Quicksilver SKUs

### CODE REVIEW

File: device/common/profiles/th5/gen/BALANCED/pg_profile_lookup.ini [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# PG lossless profiles.
+# speed cable size    xon  xoff threshold xon_offset
+  10000  5m   1248    2288 35776   0        2288
+  25000  5m   1248    2288 53248   0        2288
+  40000  5m   1248    2288 66560   0        2288
+  50000  5m   1248    2288 90272   0        2288
+ 100000  5m   1248    2288 165568  0        2288
+ 400000  5m   1248    2288 165568  0        2288
```
Comment (kperumalbfn) [inline device/common/profiles/th5/gen/BALANCED/pg_profile_lookup.ini:8]: PG profile configs for 800G?

Comment (lguohan) [reply device/common/profiles/th5/gen/BALANCED/pg_profile_lookup.ini:8]: 800G is not needed for now.

Comment (lguohan) [reply device/common/profiles/th5/gen/BALANCED/pg_profile_lookup.ini:8]: they can add 800G later.

**Comment by r12f** [comment]
tested on testbed, the configuration works and links can be up.

**Review by r12f — APPROVED** [review]
lgtm

**Comment by ccroy-arista** [comment]
@r12f @rick-arista it looks like this PR needs to be cherry-picked to 202405 now as well.

**Comment by ccroy-arista** [comment]
Awaiting cherry-pick to 202405.

**Pull request merged**